File: 006/prob6.py
#Project Euler Problem 6
#What is the difference between the sum of the squares and the square of the sums?

n = 100
sum_of_sqr = 0
sqr_of_sum = 0

#find the sum of squares up to n
for i in range(1, n + 1): sum_of_sqr += i**2

#find square of sum up to n
for i in range(1, n + 1): sqr_of_sum += i
sqr_of_sum **= 2

#display the difference
print sqr_of_sum - sum_of_sqr

File: 006/prob6.c
//Find the difference between the sum of the squares of the 
//first one hundred natural numbers and the square of the sum.
#include <stdio.h>
#include <math.h>

#define max 100

int main()
{
	int sumOfSquares = 0;
	int squareOfSums = 0;

	//find the sum of the squares
	int i = 1;

	while (i <= max)
	{
		sumOfSquares = sumOfSquares + pow(i, 2);
		squareOfSums += i;
		i++;
	}

	squareOfSums = squareOfSums * squareOfSums;

	printf("%d - %d = %d\n",squareOfSums,sumOfSquares,squareOfSums - sumOfSquares);

	return 0;
}
